ccd-definition/CaseField.json

```diff
@@ -262,17 +262,18 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "standardDirectionsLabel",
-    "Label": "Upload standard directions.",
-    "FieldType": "Label",
+    "ID": "standardDirections",
+    "Label": "1. Standard directions",
+    "FieldType": "StandardDirections",
     "SecurityClassification": "Public"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "standardDirections",
-    "Label": "Upload a file",
-    "FieldType": "Document",
+    "ID": "standardDirectionsOther",
```

Comment:
Please rename to `otherCourtAdminDocuments`. That name is agreed with Jason. A `standardDirectionsOther` does not make much sense because C6 and C9 are not standard directions.